hera_qm/ant_metrics.py

```diff
@@ -123,21 +133,33 @@ def mean_Vij_metrics(data, pols, antpols, ants, bls,
                      xants=[], rawMetric=False):
     """Calculate how an antennas's average |Vij| deviates from others.
 
-    Arguments:
-        data: data for all polarizations
-                format that must support data[i,j,pol]
-        pols : List of visibility polarizations (e.g. ['xx','xy','yx','yy']).
-        antpols: List of antenna polarizations (e.g. ['x', 'y'])
-        ants: List of all antenna indices.
-        bls: List of tuples of antenna pairs.
-        xants: List of antennas ithat should be ignored.
-               format: (ant,antpol)
-        rawMetric:return the raw mean Vij metric instead of the modified z-score
-
-    Returns:
-        meanMetrics: Dictionary indexed by (ant,antpol) of the modified z-score
-                     of the mean of the absolute value of all visibilities associated with an antenna.
-                     Very small or very large numbers are probably bad antennas.
+    Parameters
+    ----------
+    data
+        Data for all polarizations, stored in a format that supports
+        indexing via data[i,j,pol].
+    pols
+        List of visibility polarizations (e.g. ['xx','xy','yx','yy']).
+    antpols
+        List of antenna polarizations (e.g. ['x', 'y']).
+    ants
+        List of all antenna indices.
+    bls
+        List of tuples of antenna pairs.
+    xants
```

Comment:
https://github.com/HERA-Team/hera_qm/blob/7347c53d14d1cb6706e3d81c16102d494f90c6e3/hera_qm/ant_metrics.py#L138-L151

We are missing type definitions in the numpy style for these lines. (eg ` pols : list`) but at the same time it is quite obvious that it expects a list.